.github/workflows/bazel.yml

```
@@ -56,10 +56,11 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - { name: "Linux (native)",  os: ubuntu-24.04,   action: "test",  config: "--config=linux",    }
-          - { name: "Linux (systemcore)",  os: ubuntu-24.04,   action: "build",  config: "--config=systemcore",    }
+          - { name: "Linux (native)",      os: ubuntu-24.04,   container: wpilib/ubuntu-base:22.04, action: "test",   config: "--config=linux",    }
+          - { name: "Linux (systemcore)",  os: ubuntu-24.04,   container: wpilib/ubuntu-base:22.04, action: "build",  config: "--config=systemcore",    }
```

Fixing the invalid YAML:
```suggestion
          - { name: "Linux (native)",      os: ubuntu-24.04,   container: "wpilib/ubuntu-base:22.04", action: "test",   config: "--config=linux",    }
          - { name: "Linux (systemcore)",  os: ubuntu-24.04,   container: "wpilib/ubuntu-base:22.04", action: "build",  config: "--config=systemcore",    }
```